config/settings.yml

```diff
@@ -1405,6 +1405,10 @@ vanotify:
     connected_applications: https://www.va.gov/profile/connected-applications
     password_reset: https://www.va.gov/resources/signing-in-to-vagov/#what-if-i-cant-sign-in-to-vago
 
+va_notify:
+  status_callback:
+    bearer_token: fake_bearer_token
```

Comment:
I'm just now noticing there is already a VA Notify section. Search for `vanotify:`. I think it makes sense to put these vanotify configs in the same section as the other vanotify settings.

Also, here is [the documentation](https://depo-platform-documentation.scrollhelp.site/developer-docs/vets-api-on-eks#VetsAPIonEKS-Secretvalues) on adding this to the manifest repo.

Reply:
I have updated `vanotify:` section and added this variable into the manifest repo:
https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/3183

can you please take a look at above PR and let me know if there is anything I need to work?